core/domain/collection_domain.py

```diff
@@ -907,132 +912,71 @@ def exploration_ids(self):
 
     @property
     def init_exploration_ids(self):
-        """Returns a list of exploration IDs that are starting points for this
-        collection (ie, they require no prior skills to complete). The order
-        of these IDs is given by the order each respective exploration was
-        added to the collection.
+        """Returns the first element in the node list of the collection, which
+           corresponds to the first node that the user would encounter.
 
         Returns:
-            list(str). List of exploration IDs.
+            list(str). A single element list of the exploration ID of the
+                first node.
         """
-        init_exp_ids = []
-        for node in self.nodes:
-            if not node.prerequisite_skill_ids:
-                init_exp_ids.append(node.exploration_id)
-        return init_exp_ids
-
-    def get_next_exploration_ids(self, completed_exploration_ids):
-        """Returns a list of exploration IDs for which the prerequisite skills
-        are satisfied. These are the next explorations to complete for a user.
-        If the list returned is empty and the collection is valid, then all
-        skills have been acquired and the collection is completed. If the input
-        list is empty, then only explorations with no prerequisite skills are
-        returned. The order of the exploration IDs is given by the order in
-        which each exploration was added to the collection.
+        if len(self.nodes) > 0:
+            return [self.nodes[0].exploration_id]
+        else:
+            return []
+
+    def get_next_exploration_id(self, completed_exp_ids):
+        """Returns the first exploration id in the collection that has not yet
+           been completed by the learner.
 
         Args:
             completed_exploration_ids: list(str). List of completed exploration
                 ids.
 
         Returns:
-            list(str). A list of exploration IDs for which the prerequisite
-            skills are satisfied.
+            list(str). A single element list of the next exploration id
+                in the node list
         """
-        acquired_skill_ids = self.get_acquired_skill_ids_from_exploration_ids(
-            completed_exploration_ids)
-        next_exp_ids = []
-        for node in self.nodes:
-            if node.exploration_id in completed_exploration_ids:
-                continue
-            prereq_skill_ids = set(node.prerequisite_skill_ids)
-            if prereq_skill_ids <= acquired_skill_ids:
-                next_exp_ids.append(node.exploration_id)
-        return next_exp_ids
+        for exp_id in self.exploration_ids:
+            if exp_id not in completed_exp_ids:
+                return [exp_id]
+        return []
 
     def get_next_exploration_ids_in_sequence(self, current_exploration_id):
```

Comment:
Decide whether this returns one thing or multiple things, and adjust the name and return type accordingly.